README.MD

```diff
@@ -285,7 +314,18 @@ Please note these demos should be considered as CSS "Proofs of Concepts". They m
 
 **[⬆ back to top](#quick-links)**
 
-***
+---
+
+&nbsp;
+
+## <a id="TypewriterEffect"></a>Typewriter Effect
+
+[<img src="images/typewriter-effect.gif" height="230" title="Typewriter Effect">](https://codepen.io/hrshainik/pen/PopRWGN)
+
+**[⬆ back to top](#quick-links)**
+
+---
+
```

Comment:
this looks cool!
if you can add the html code in `example` folder and rebase from latest master, we can merge it. 
thanks!

Reply:
Hi Robert Chang,
Thank you for your feedback and for considering my pull request. I've already added the HTML code to the example folder as requested. However, during the rebase process, I encountered some conflicts that need to be resolved before the pull request can be merged.

I would appreciate it if you could take a look at the conflicts and help me resolve them. Once the conflicts are resolved, I believe the pull request will be ready for merging into the master branch.